gui/cache.go

```diff
@@ -45,10 +45,10 @@ type rewardQuota struct {
 // pendingPayment represents an unpaid reward payment. It is json annotated so
 // it can easily be encoded and sent over a websocket or pagination request.
 type pendingPayment struct {
-	WorkHeight    string `json:"workheight"`
-	WorkHeightURL string `json:"workheighturl"`
-	Amount        string `json:"amount"`
-	CreatedOn     string `json:"createdon"`
+	WorkHeight             string `json:"workheight"`
```

Comment:
Just noticing this, why aren't `WorkHeight`, `Amount` and `EstimatedPaymentHeight` their default types here? I reckon the templates could have formatting needed to cast the types when needed.

Reply:
If we rely upon the templates to execute the formatting logic then it will be performed on every single field, every single time a page is loaded.

The formatting is currently executed before the data is inserted into the cache, which means it is only performed once and never duplicated unnecessarily.

Reply:
noted, thanks.